packages/@lightningjs/ui-components/src/docs/constants.js

```diff
@@ -16,57 +16,6 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-export const CATEGORIES = {
```

Comment:
Story sorting is different v7, sorting is statically analyzed
Important Note: storySort must be a self-contained function that does not reference external variables
Please reference [this section](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#v7-style-story-sort) of the migration guide